### ISSUE TITLE
Missing options in  VS 2022 (and worakaround)

### ISSUE THREAD

**Issue by renestein** [renestein/ConfigureAwaitEnforcer#8]
The ConfigureAwaitEnforcer options don't appear in the VS2022 Options tree. Is there another way to set the "Diagnostic severity" setting? 

From https://marketplace.visualstudio.com/items?itemName=Rene-Stein.ConfigureAwaitEnforcer2022&ssr=false#qna

**Comment by renestein** [comment]
Hi,
and thanks for the report.
It is strange that the same code works in VS 2019, but not in VS 2022. I hoped that this is an error in the preview version of VS 2022.

Anyway:

WORKAROUND:
The config is stored in `%LOCALAPPDATA%\RStein.ConfigureAwaitEnforcer\ConfigureAwaitEnforcerProperties`

e. g. the full path is:
`c:\Users\rene\AppData\Local\RStein.ConfigureAwaitEnforcer\ConfigureAwaitEnforcerProperties
`
The file ConfigureAwaitEnforcesProperies should use UTF-8 encoding and have the following content.
`Diagnostics_Severity=Info`

(Info, Hidden, Warning, Error) are valid values.

**Comment by tigrom** [comment]
Hi, `ConfigureAwaitEnforcerProperties` file isn't to be found, in fact, the entire `%LOCALAPPDATA%\user.ConfigureAwaitEnforcer\` missing completly from the system 
(VS2022 v17.10.5)

The problem is that the severity is set to `Error` by default, so the only way to bypass the error is to set the property OR remove the extension. 
I suggest that severity should not be set to `Error` by default.

**Comment by renestein** [comment]
Hi,
the path is _%LOCALAPPDATA%\RStein.ConfigureAwaitEnforcer\ConfigureAwaitEnforcerProperties_.
For example.
c:\Users\tigran\AppData\Local\RStein.ConfigureAwaitEnforcer\ConfigureAwaitEnforcerProperties .
My system.
Visual studio 2022 17.13.0. Preview 1.0
Thanks for the recommendation.

**Comment by tigrom** [comment]
Hey - thanks for the quick reply.
The problem is the folder is missing from the system.
I've searched for it (in depth).

`%LOCALAPPDATA%\{username}.ConfigureAwaitEnforcer`

Maybe I can manually create this folder, and the corresponding `ConfigureAwaitEnforcerProperties`, but I don't know its structure.

**Comment by renestein** [comment]
This folder does not exist.
{username}.ConfigureAwaitEnforcer.
Is possible to search in folder %LOCALAPPDATA%\ for  RStein.ConfigureAwaitEnforcer?

**Comment by tigrom** [comment]
Yes - it's just an example with {username} placeholder. Of course I've used my username. 
I tried anything with `ConfigureAwaitEnforcer` in my search
.
There is nothing found on the system other then the extension folder `C:\Users\vladvoic\AppData\Local\Microsoft\VisualStudio\17.0_b8872b4c\Extensions\g0wklkzp.rch`

![image](https://github.com/user-attachments/assets/667bdaa6-1bec-4d38-ba49-93048d475a55)

I also used a tool to search with wildcards, this is the only things it finds on the entire hard drive:

![image](https://github.com/user-attachments/assets/baab6be6-e4df-4e8c-b629-43bf3f622549)

It's definitely missing from my system, but maybe I can manually create it.

**Comment by renestein** [comment]
ConfigureAwaitEnforcerProperties file is simple. However, it must be in the expected location.

```c#
Diagnostics_Severity=Error
``` 

Config must be here.
```c#

  public const string CONFIG_NAME = "ConfigureAwaitEnforcerProperties";
   public const string EXTENSION_NAME = "RStein.ConfigureAwaitEnforcer";
    
 return Path.Combine(Environment.GetEnvironmentVariable("LOCALAPPDATA"),
                          EXTENSION_NAME,
                          CONFIG_NAME);
``` 
Is possible that your "LOCALAPPDATA" was redirected to another location? Could you please evaluate the path above in the code? Are you running VS under your account, and not in "admin" (elevated) mode?

**Comment by tigrom** [comment]
I manually create this file & folder and it seems to work! Thank you so much

**Comment by renestein** [comment]
I am glad that it works. Thanks for your report and sorry for the inconvenience.
